app/poros/movie.rb

```diff
@@ -1,5 +1,5 @@
 class Movie
-  attr_reader :title, :vote_average, :id, :summary, :reviews
+  attr_reader :title, :vote_average, :id, :summary, :reviews, :runtime
```

Comment:
It totally makes sense that we need to add this now that we are calling it in the form.